sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/DataSourceV2AnalysisSuite.scala

```diff
@@ -446,6 +452,27 @@ abstract class DataSourceV2AnalysisSuite extends AnalysisTest {
       "Cannot safely cast", "'x'", "DoubleType to FloatType"))
   }
 
+  test("bypass output column resolution") {
+    val table = TestRelationAcceptAnySchema(StructType(Seq(
+      StructField("a", FloatType, nullable = false),
+      StructField("b", DoubleType))).toAttributes)
+
+    val query = TestRelation(StructType(Seq(
+      StructField("s", StringType))).toAttributes)
+
+    withClue("byName") {
+      val parsedPlan = byName(table, query)
+      assertResolved(parsedPlan)
+      checkAnalysis(parsedPlan, parsedPlan)
+    }
+
+    withClue("byPosition") {
```

Comment:
@rdblue if 2 tests are very similar, it's recommended to use `withClue` and merge the 2 tests into one. For example, when the byPosition case fails, we will see
```
bypass output column resolution *** FAILED *** (36 milliseconds)
[info]   byPosition (DataSourceV2AnalysisSuite.scala:473)
[info]   org.scalatest.exceptions.TestFailedException:
...
```

Reply:
I'll send a followup PR to update the entire test suite to use `withClue` later.

Reply:
Okay, sounds good.